src/components/bookItem.js

```diff
@@ -39,14 +43,29 @@ const Books = (props) => {
           </button>
         </div>
       </div>
+      <div className="status">
+        <img src={spinner} alt="progression circle" className="spinner-img" />
+        <div>
+          <p className="percentage">0%</p>
+          <p className="status-details">Incomplete</p>
+        </div>
+      </div>
+      <section className="position">
+        <h2 className="pos-header">CURRENT CHAPTER</h2>
+        <p className="pos-details">Chapter 1</p>
+        <div>
+          <button type="button" name="update" id="update">UPDATE PREGRESS</button>
+        </div>
+      </section>
```

Comment:
- [ ] Kindly know that this milestone is to help you improve your styling, currently you have just skipped just few things to do, kindly see the images I pasted below and fix them
your design :point_down: 
![WhatsApp Image 2022-11-11 at 1 48 24 PM](https://user-images.githubusercontent.com/66097371/201344113-f91357b3-ffa0-49b4-a2c9-345d47fee5ee.jpeg)
there is a border between the status and the button which is currently missing in your design, kindly add that
zeplin design :point_down: 
![Screenshot from 2022-11-11 13-54-30](https://user-images.githubusercontent.com/66097371/201344801-2f416051-9c0b-4495-bf9f-2bf4197306ac.png)